README.md

```diff
@@ -1,70 +1,69 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Metrics-webapp 
 
-## Available Scripts
+Covid-19 Tracker is a mobile web application that provide the user the new cases of Covid-19 for each continent and individual coutries for that particular continent.
 
-In the project directory, you can run:
+## Built With
 
-### `npm start`
+- ReactJS
+- Redux and react-redux
+- HTML
+- CSS
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+![screenshot](Coming soon)
 
-### `npm test`
+## Live Demo 
+[Live Demo](https://tobinscovid-19-tracker.herokuapp.com/)
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
 
-### `npm run build`
+## Presentation Video
+[Presentation Video](https://www.loom.com/share/56da292ac7744509baab0d2bc189985b)
```

Comment:
[Optional] As required, the video should contain an highlight about a piece of the code you find interesting or proud of making.